cmd/itzo-launcher/itzo-launcher.go

```diff
@@ -99,16 +100,33 @@ func EnsureItzo() (string, error) {
 }
 
 func RunItzo(itzoPath string) error {
+	usePodman := false
+	config, err := readCellConfig()
+	if err != nil {
+		// ignore
+	}
+	klog.Info(config)
+	if usePodmanFlag, ok := config["usePodman"]; ok {
```

Comment:
Instead of hardcoding the conversion logic for each flag one by one, how about we add a convention. Example: any key in the cell config map that starts with `itzo-flag` will be added as a flag when launching itzo, i.e. `itzo-flag-v: 3` and `itzo-flag-use-podman: true` will start `itzo --v=3 --use-podman=true`.

Reply:
I like the idea, however, I'm wondering whether we abstract the way we pass the flags to itzo after just one case when we need it :) I usually try to follow YAGNI in such cases. Do you think it is likely that we will need to pass other flags to itzo  in the foreseeable future? If so, I can make it more generic. Otherwise, I may leave it as it is and add a todo comment.